src/reducers/breakpoints.js

```diff
@@ -56,6 +56,12 @@ export function makeLocationId(location: Location) {
   return `${sourceId}:${line}:${column}`;
 }
 
+export function makePendingLocationId(location: Location) {
+  let { sourceUrl, line, column } = location;
+  column = column || "";
```

Comment:
since this is the only reassigned variable and since we are not using es6 object shorthand we can instead use consts and make this a new variable like `const columnString = column || "";`